lua/neogit/popups/log_margin/init.lua

```diff
@@ -0,0 +1,26 @@
+local popup = require("neogit.lib.popup")
+
+local M = {}
+
+function M.create()
+  local p = popup
+    .builder()
+    :name("NeogitLogMarginPopup")
+    :switch("n", "max-count", "Prune deleted branches")
```

Comment:
Quick note: switches are boolean - cli flag with no value. `option`s capture user input. There should be an example on the log popup of an option with preset choices you can cycle through.

Reply:
Gotcha, updated to the correct builder args I think.